lib/sync/sync-server.js

```diff
@@ -45,10 +45,16 @@ var syncWorker;
 var DEFAULT_SYNC_CONF = {
   /** @type {Number} how often pending workers should check for the next job, in ms. Default: 500 */
   pendingWorkerInterval: 500,
+  /** @type {Object} the backoff strategy for the pending worker to use. Default is exponetial, and the max delay is 60s */
```

Comment:
using 'max delay' in this comment makes it sound like you can't set the max over 60s.

Maybe something like this could read better?
'The default strategy is "exp" with a max delay of 60 seconds'